pkg/reconciler/pipelinerun/resources/pipelineref.go

```diff
@@ -118,15 +118,9 @@ func GetVerifiedPipelineFunc(ctx context.Context, k8s kubernetes.Interface, tekt
 			source = s.URI
 		}
 		logger := logging.FromContext(ctx)
-		if config.CheckEnforceResourceVerificationMode(ctx) || config.CheckWarnResourceVerificationMode(ctx) {
-			if err := trustedresources.VerifyPipeline(ctx, p, k8s, source, verificationpolicies); err != nil {
-				if config.CheckEnforceResourceVerificationMode(ctx) {
-					logger.Errorf("GetVerifiedPipelineFunc failed: %v", err)
-					return nil, nil, fmt.Errorf("GetVerifiedPipelineFunc failed: %w: %v", trustedresources.ErrResourceVerificationFailed, err)
-				}
-				logger.Warnf("GetVerifiedPipelineFunc failed: %v", err)
-				return p, s, nil
-			}
+		if err := trustedresources.VerifyPipeline(ctx, p, k8s, source, verificationpolicies); err != nil {
+			logger.Errorf("GetVerifiedPipelineFunc failed: %v", err)
```

Comment:
https://google.github.io/styleguide/go/best-practices.html#logging-errors
"Avoid duplication. If you return an error, it’s usually better not to log it yourself but rather let the caller handle it."